apps/onnx/model.cpp

```diff
@@ -0,0 +1,549 @@
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "common_types.h"
+#include "onnx_converter.h"
+#include <fstream>
+#include <random>
+#include <sys/time.h>
+#include <unordered_set>
+
+namespace py = pybind11;
+
+static HalideModel convert_onnx_model(
```

Comment:
here and elsewere prefer wrapping local functions in an anonymous namespace to "static" declarations